src/api/category/model/category-select.model.ts

```diff
@@ -0,0 +1,11 @@
+interface CategoryPrismaSelect {
+  id?: boolean;
+  uuid?: boolean;
+  name?: boolean;
+  products?: boolean;
```

Comment:
For this type of entries, relational entries, i think instead of boolean we can use the ProductSelect interface located in the model folder of each entity